File: CPP/Petshop.cpp
#include <iostream>
#include <string>

using namespace std;

class Petshop
{
private:
    string id, nama, kategori;
    int harga;
public:
    Petshop(){
        this->id = "";
        this->nama = "";
        this->kategori = "";
        this->harga = 0;
    };
    Petshop(string id, string nama, string kategori, int harga){
        this->id = id;
        this->nama = nama;
        this->kategori = kategori;
        this->harga = harga;
    }

    void set_id(string id){
        this->id = id;
    }
    void set_nama(string nama){
        this->nama = nama;
    }
    void set_kategori(string kategori){
        this->kategori = kategori;
    }
    void set_harga(int harga){
        this->harga = harga;
    }

    string get_id(){
        return this->id;
    }
    string get_nama(){
        return this->nama;
    }
    string get_kategori(){
        return this->kategori;
    }
    int get_harga(){
        return this->harga;
    }

    ~Petshop(){
    };
};

File: JAVA/Petshop.java
public class Petshop {

    private String id, nama, kategori;
    private int harga;

    public Petshop(){
    }

    public String getId() {
        return id;
    }
    public void setId(String id) {
        this.id = id;
    }
    public String getNama() {
        return nama;
    }
    public void setNama(String nama) {
        this.nama = nama;
    }
    public String getKategori() {
        return kategori;
    }
    public void setKategori(String kategori) {
        this.kategori = kategori;
    }
    public int getHarga() {
        return harga;
    }
    public void setHarga(int harga) {
        this.harga = harga;
    }

    public void setAll(String id, String nama, String kategori, int harga){
        this.id = id;
        this.nama = nama;
        this.kategori = kategori;
        this.harga = harga;
    }
}